TTBot/Commands/ChampionshipsModule.cs

```diff
@@ -179,41 +181,81 @@ public async Task GetStandings([Remainder] string args = null)
-                            sb.AppendLine("```");
-                            while ((line = reader.ReadLine()) != null)
+                            graphics.DrawString(
+                                championship,
+                                championship.Trim().Length < 10 ? championshipFont : smallerChampionshipFont,
+                                new SolidBrush(Color.FromArgb(213, 213, 213)),
+                                championshipX,
+                                championshipY);
+
+                            int y = posYStart;
+                            foreach (ChampionshipResultsModel r in orderedResults)
                             {
-                                if (i > 0 && i%10 == 0)
-                                {
-                                    sb.AppendLine("```");
-                                }
-                                sb.AppendLine(line);
-                                if (i % 10 == 9)
-                                {
-                                    sb.AppendLine("```");
-                                    await ReplyAsync(sb.ToString());
-                                    sb = new StringBuilder();
-                                }
-                                i++;
+                                graphics.FillRoundedRectangle(Brushes.White, posXStart, y - 5, 60, 40, 4);
+
+                                var posX = r.Pos <= 9
+                                    ? posXStart + 15
+                                    : posXStart + 5;
+
+                                graphics.DrawString(r.Pos.ToString(), numberFont, Brushes.Black, posX, y);
+                                graphics.DrawString(
+                                    r.Driver,
+                                    r.Driver.Length <= 25 ? font : longDriverFont,
+                                    Brushes.White,
+                                    driverX,
+                                     r.Driver.Length <= 25 ? y : y + 6);
+                                graphics.DrawString(r.Number, font, Brushes.White, numberX, y);
+                                graphics.DrawString(r.Points, font, Brushes.White, pointsX, y);
+                                graphics.DrawString(r.Diff, font, Brushes.White, diffX, y);
+
+                                lastRowY = y;
+
+                                y += 50;
                             }
-                        }
+                        }   
+                    }
+
+                    var standingsFilePath = $"{templateFilePath.Replace(".png", "")}-{Guid.NewGuid()}.png";
+                    if (lastRowY + 75 < image.Height)
+                    {
+                        var imageCropRect = new Rectangle(0, 0, image.Width, lastRowY + 75);
+                        var croppedImage = (Bitmap)image.Clone(imageCropRect, System.Drawing.Imaging.PixelFormat.Format32bppArgb);
+                        croppedImage.Save(standingsFilePath);
```

Comment:
Is there no API to do this from a memory stream?

Reply:
https://github.com/Still34/Discord.Net/blob/docs/pre-release/src/Discord.Net.WebSocket/Entities/Channels/ISocketMessageChannel.cs/#L88 this overload allows you pass a stream which will remove the need to save to disk.

Reply:
Neat. Will use that